.github/assignment.yml

```diff
@@ -0,0 +1,49 @@
+---
+- rule:
+    path: "specification/**/Microsoft.Network/**"
+    reviewers:
+      - lirenhe
+      - zikalino
+
+- rule:
+    path: "specification/**/date-plane/**"
+    reviewers:
+      - anuchandy
+      - jhendrixMSFT
+      - lmazuel
+      - tjprescott
+
+- rule:
+    path: "documentation/**"
+    reviewers:
+      - akning-ms
```

Comment:
Pls also add @josefree for Document as first reviewer

Reply:
So it will rotate to assign for me and josefree? we should always assign to Josefree for documents. or assign both everytime?

Reply:
The current design is random pick.